packages/kbn-unified-field-list/src/components/field_popover/field_popover_header.tsx

```diff
@@ -90,6 +101,24 @@ export const FieldPopoverHeader: React.FC<FieldPopoverHeaderProps> = ({
             <h5 className="eui-textBreakWord">{field.displayName}</h5>
           </EuiTitle>
         </EuiFlexItem>
+        {onAddBreakdownField && (
+          <EuiFlexItem grow={false} data-test-subj="fieldPopoverHeader_addBreakdownField">
+            <EuiToolTip
+              content={buttonAddBreakdownFieldProps?.['aria-label'] ?? addBreakdownFieldTooltip}
+            >
+              <EuiButtonIcon
+                data-test-subj={`fieldPopoverHeader_addBreakdownField-${field.name}`}
+                aria-label={addBreakdownFieldTooltip}
+                {...(buttonAddBreakdownFieldProps || {})}
+                iconType="visBarVerticalStacked"
+                onClick={() => {
+                  closePopover();
+                  onAddBreakdownField(field);
```

Comment:
We could also extend `test/functional/page_objects/unified_field_list.ts` service with this new action and add a functional test which would check that pressing the button changes the histogram state. We have some breakdown-related tests in `test/functional/apps/discover/group1/_discover_histogram_breakdown.ts` and `test/functional/apps/discover/esql/_esql_view.ts`.